alfasim_sdk/hook_specs_gui.py

```diff
@@ -19,6 +19,15 @@ def alfasim_get_additional_variables():
     """
 
 
+@hookspec
+def alfasim_get_status(ctx):
+    """
+    Allows plugins to execute custom checks on ALFAsim.
```

Comment:
One of the most important things we should document is inputs and expected outputs... feel free to use type annotations here.